assemblies/guardrails-orchestrator-service.adoc

```diff
@@ -0,0 +1,27 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+[id="guardrails_orchestrator_service_{context}"]
+= Guardrails orchestrator service
+
+[role='_abstract']
+
+The TrustyAI Guardrails Orchestrator service is a tool to invoke detections on text generation inputs and outputs, as well as standalone detections.
+
+It is underpinned by the open-source project link:https://github.com/foundation-model-stack/fms-guardrails-orchestrator[FMS-Guardrails Orchestrator] from IBM and is deployable through a Custom Resource Definition (CRD) managed by the TrustyAI Operator.
+
```

Comment:
avoid passive voice:
...from IBM. You can deploy the Guardrails Orchestrator service through a Custom Resource Definition (CRD) that is managed by the TrustyAI Operator.